routers/web/auth/webauthn.go

```diff
@@ -67,10 +67,7 @@ func WebAuthnLoginAssertion(ctx *context.Context) {
 		return
 	}
 
-	// FIXME: DEPRECATED appid is deprecated and is planned to be removed in v1.18.0
-	assertion, sessionData, err := wa.WebAuthn.BeginLogin((*wa.User)(user), webauthn.WithAssertionExtensions(protocol.AuthenticationExtensions{
-		"appid": setting.U2F.AppID,
-	}))
+	assertion, sessionData, err := wa.WebAuthn.BeginLogin((*wa.User)(user))
```

Comment:
Could we remove that `appid`?

Reply:
Which appid? It's removed in the settings module.

Reply:
As far as I know it's only required for U2F applications and Firefox+Chromium have deprecated this extension during login IIRC.